packages/manager/src/features/Account/EnableObjectStorage.tsx

```diff
@@ -88,13 +104,28 @@ export const EnableObjectStorage: React.FC<CombinedProps> = (props) => {
       .catch(handleError);
   };
 
-  const actions = () => (
+  const renderActions = (
+    disabled: boolean,
+    loading: boolean,
+    onClose: () => void,
+    onSubmit: () => void
+  ) => (
     <ActionsPanel>
-      <Button buttonType="secondary" onClick={handleClose}>
+      <Button
+        buttonType="secondary"
+        onClick={onClose}
+        data-testid={'dialog-cancel'}
+      >
         Cancel
       </Button>
 
-      <Button buttonType="primary" onClick={handleSubmit} loading={isLoading}>
+      <Button
+        buttonType="primary"
+        onClick={onSubmit}
+        disabled={disabled}
+        loading={loading}
+        data-testid={'dialog-confirm'}
+      >
         Confirm cancellation
```

Comment:
Should we capitalize this?

```suggestion
        Confirm Cancellation
```

Reply:
I was wondering about this too, so thanks for pointing it out. I went back through the app and found that generally all words are capitalized on our buttons, so I went ahead and made your suggested change here.